lang_php/parsing/cst_php.ml

```diff
@@ -261,6 +261,7 @@ and expr =
   | Lambda of lambda_def
   | ShortLambda of short_lambda_def
   (* should be a statement ... *)
+  | Echo of tok * expr comma_list
```

Comment:
I would add more comment here, saying "Echo was originally a statement in the grammar and AST,
but it simplifies things (e.g., tainting in semgrep) to have it as an expression".

Actually I'm not sure we need that, this could be done in php_to_generic.ml when we parse a pattern
that ended up being an ExprStmt (Call (builtin "echo")).

But this works too I guess.

Reply:
Hmmm, I did consider that option but then I thought that I would make it impossible to match `echo 'foo';` (including the `;`). Yet now I realize that I can also do this in Tainting_generic. Do you have a preference?

Reply:
I think we can merge this

This is ok.